src/transformers/models/autoformer/configuration_autoformer.py

```diff
@@ -0,0 +1,249 @@
+# coding=utf-8
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
```suggestion
```
(nit) Typically only attribution is given in the modeling file as the config is HF specific